### PR TITLE
Update personality

### PR DESCRIPTION
**Pull request by ifndefdeadmau5**
Now it gets the data from server at startup and apply to slider value.

image

### CODE REVIEW

**Comment by MechanicKim** [comment]
I think @ifndefdeadmau5 is good at using ES6, ES7. It might be so good to hold a brown bag session about ES6, ES7 for everyone!

@SarahhhC Please review it!

File: src/components/SignUp/Personality.js [hunk 1]
```diff
@@ -45,10 +45,44 @@ class Personality extends Component {
...
     }
   }
 
+  /**
+   * Return -1 if given personality is from left side, or return 1
+   * if given personality is from right side.
+   * @param {string} option - given personality
+   * @returns {number} The direction. if nothing matches, return 0
+   */
+  findDirection(option) {
```
Comment (SarahhhC) [inline src/components/SignUp/Personality.js:72]: Can you explain a little about findDirection?

Comment (ifndefdeadmau5) [reply src/components/SignUp/Personality.js:72]: Sure. First, the range of personality is basically -2 through 2 and 0 is indeed neutral value.
Second, option[0] goes left side and option[1] goes right side.
This function takes string of personality and find it's origin(from left side or right side) in order to figure out which way slider thumb should go.

**Comment by MechanicKim** [comment]
Please update branch @ifndefdeadmau5.
After that, please give him 'Looks good to me'. @SarahhhC.

File: src/components/SignUp/Personality.js (hunk 1 above)
Comment (SarahhhC) [reply src/components/SignUp/Personality.js:72]: Thanks!

**Comment by SarahhhC** [comment]
Thanks @ifndefdeadmau5 LGTM

**Pull request merged**